sys/include/net/ng_ipv6/netif.h

```diff
@@ -87,7 +107,45 @@ extern "C" {
  * @{
  * @name Flags for the interfaces
  */
-#define NG_IPV6_NETIF_FLAGS_SIXLOWPAN   (0x01)  /**< interface is 6LoWPAN interface */
+/**
+ * @brief   Interface is 6LoWPAN interface.
+ */
+#define NG_IPV6_NETIF_FLAGS_SIXLOWPAN           (0x0001)
+
+/**
+ * @brief   Flag to indicate that routing is enabled on the interface.
+ */
+#define NG_IPV6_NETIF_FLAGS_ROUTER              (0x0002)
+
+/**
+ * @brief   Flag to indicate that the interface sends periodic router
+ *          advertisements and in response to router solicitations.
+ */
+#define NG_IPV6_NETIF_FLAGS_RTR_ADV             (0x0004)
+
+/**
+ * @brief   Flag to indicate that ng_ipv6_netif_t::mtu shall be propagated
+ *          with the MTU options in router advertisements.
+ */
+#define NG_IPV6_NETIF_FLAGS_ADV_MTU             (0x0008)
+
+/**
+ * @brief   Flag to indicate that ng_ipv6_netif_t::cur_hl shall be propagated
+ *          in router advertisements.
+ */
+#define NG_IPV6_NETIF_FLAGS_ADV_CUR_HL          (0x0010)
+
+/**
+ * @brief   Flag to indicate that the interface has other address
+ *          configuration.
+ */
+#define NG_IPV6_NETIF_FLAGS_OTHER_CONF          (0x4000)
+
+/**
+ * @brief   Flag to indicate that the interface has managed address
+ *          configuration.
```

Comment:
The plan was to reference the flag when the header was included, but since it is not here yet: sure.